reference/Counter/CSharp-MVUX/Directory.Packages.props

```diff
@@ -1,16 +1,16 @@
 ﻿<Project ToolsVersion="15.0">
   <ItemGroup>
-    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
     <PackageVersion Include="Microsoft.Windows.Compatibility" Version="8.0.0-rc.2.23479.10" />
     <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.4.231008000" />
```

Comment:
Are these package bumps needed for something?

Reply:
Uno.Extensions.Reactive 4.0.0 was needed for UpdateAsync, we only had Update before